sshmenu/sshmenu.py

```diff
@@ -45,16 +61,23 @@ def display_menu(targets):
         puts('Whoops, you don\'t have any targets listed in your config!')
         exit(0)
 
-    # Determine the longest host length
+    # Determine the longest host and line length
     longest_host = -1
-    for target in targets:
+    longest_line = -1
+    for index, target in enumerate(targets):
         length = len(target['host'])
+        # Check host length
         if length > longest_host:
             longest_host = length
 
-    # Clear screen, give instructions
+        # Check line length
+        desc = '%2d ' % (index) + target['host'].ljust(longest_host) + ' | ' + target['friendly']
```

Comment:
I think this should be split into a separate function, like `def get_target_desc(target):` in order to avoid this line being duplicated further down.

Another alternative is to store the value in target['desc'] and reference that value further down.